Backend/AlgorithmsManager.py

```diff
@@ -474,20 +474,42 @@ def get_most_frequent_retweeters(self, username, num_tweets=20, num_retweets=20)
 
         return res
 
+
+
     def get_network_tweets_text(self, id2, username, count, depth):
```

Comment:
this unfortunately conflicts with changes made in the most recent branch merged into master (making the friends-vs-followers toggle on the word cloud page work). Would you please modify this code to mimic those changes to this function?